tumblr-photo-video-ripper.py

```diff
@@ -55,12 +55,12 @@ def _handle_medium_url(self, medium_type, post):
                 video_player = post["video-player"][1]["#text"]
                 hd_pattern = re.compile(r'.*"hdUrl":("([^\s,]*)"|false),')
                 hd_match = hd_pattern.match(video_player)
-                if hd_match is not None:
+                if hd_match.group(2) is not None:
```

Comment:
Since we only use `hd_match.group(2)` here, I think it is fine to still use `hd_match`.

Also `hd_match.group(2)` will raise exceptions if `hd_match` is None.

Combined the above reasons, I prefer to stay unchanged `hd_match`.